cellprofiler/modules/identifyobjectsmanually.py

```diff
@@ -28,7 +30,14 @@
 YES          NO           NO
 ============ ============ ===============
 
-"""
+See also
+^^^^^^^^
+
+{HELP_ON_SAVING_OUTLINES}
+
+""".format(**{
+    "HELP_ON_SAVING_OUTLINES": _help.HELP_ON_SAVING_OUTLINES
```

Comment:
`"HELP_ON_SAVING_OBJECTS": _help.HELP_ON_SAVING_OBJECTS`

Reply:
I don't know how I missed this! :)